games-arcade/openclaw/openclaw-1.0.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A 2D side-scroller platform video game"
+DESCRIPTION="
+Captain Claw is a 2D side-scroller platform video game \
+published by Monolith Productions in 1997 about the anthropomorphic \
+pirate cat Captain Nathaniel Joseph Claw, who sets on a quest to find \
+an ancient amulet while fighting enemies and solving puzzles."
+HOMEPAGE="https://www.lith.com/games/captain-claw"
+COPYRIGHT="2012 Michael L. McShaffry and David Graham"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="150f7edb7542f3f877b42afc948cc24ea9a46c48"
+SOURCE_URI="https://github.com/pjasicek/OpenClaw/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9521811f572ca6c61288628cc32ff86f9c7a455afa09387232282f936a531845"
+SOURCE_DIR="OpenClaw-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
It doesn't build for x86_gcc2 so suggestion:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Reply:
Also, I would set the other architectures as "?" (even the SECONDARY_ARCHITECTURES as you need the data to run the binary)